app/PropositionFactory.php

```diff
@@ -67,6 +71,9 @@ public function getFlagCount($id) {
 	public function getFlagTypeCount($id, $type) {
 		return Flags::where('proposition', [$id])->where('type', $type)->count();
 	}
+	public function getGlobalFlagCount() {
+		return Flags::count();
```

Comment:
Yes, currently fixing the bug, needed to join with propositions model to filter proposition status. My bad ...